app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -63,7 +63,7 @@ class FeedFragment : BaseListFragment<FeedState, Unit>() {
 
     private var groupId = FeedGroupEntity.GROUP_ALL_ID
     private var groupName = ""
-    private var oldestSubscriptionUpdate: Calendar? = null
+    private var oldestSubscriptionUpdate: OffsetDateTime? = null
```

Comment:
Could you add some reasoning behind the "why" of this change. Maybe add it to the commit message too

I assume it's because of that tests with `Calendar`, that is weird.

Reply:
Same as above.

Reply:
But why was the change made of adding that method?

Reply:
Previously, PrettyTime had no support for `java.time`: https://github.com/ocpsoft/prettytime/issues/109

Reply:
Then please add that information to the commit message